crates/relayer/src/chain/ckb4ibc.rs

```diff
@@ -668,6 +677,7 @@ impl ChainEndpoint for Ckb4IbcChain {
         _target: Height,
         _client_state: &AnyClientState,
     ) -> Result<Self::LightBlock, Error> {
+        // use default is ok
```

Comment:
Why use default is ok?

Reply:
because this method doesn't make sense for now

Reply:
The comment should explain the reason, not only describe the behavior.